browser-sign-in/package.json

```diff
@@ -10,25 +10,25 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@okta/okta-react-native": "^1.4.0",
-    "events": "^3.1.0",
-    "react": "16.8.6",
-    "react-native": "^0.60.0"
+    "@okta/okta-react-native": "^1.4.4",
+    "events": "^3.2.0",
+    "react": "16.13.1",
+    "react-native": "0.60.0"
```

Comment:
Any specific reason to lock on v0.60?

Reply:
Good catch, will go through my browser history from when I worked on this to find out why I locked it to that or if it was just a typo

Reply:
It was because of this warning below, tried to cover all of the bases when I was troubleshooting 
`npm WARN @okta/okta-react-native@1.4.4 requires a peer of react-native@^0.60.0 but none is installed. You must install peer dependencies yourself.`

Luckily, it's actually not necessary to lock the version. I just recreated the entire project with the latest versions of both react + react-native using [this](https://react-native-community.github.io/upgrade-helper/?from=0.60.0&to=0.63.3 and its running without issues) guide, since I needed a newer react-native version for a CIAM project. 

Is there interest in a PR with an update to the latest react + react native? If you'd prefer to stay with the version you already had, I can just redo the PR I made earlier without the react native version locking and leave it at the original version, but also keeping the fixes I made to get building to work.

Reply:
I think we can remove the lock, and keep the `^` for the version, which should be able to install the latest non-breaking version for this sample. 
Ref: https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004